### PR TITLE
Feature/sprint2/story1 view top 10 movie titles on netflix

### PR DESCRIPTION
**Pull request by varoongupta**
Story 1
This updates and displays information about the movie:
year released, runtime, rating, genre, synopsis

Please review swiping-container.js changes

Note:
1. Some data wihtin the description/synopsis appears as an error (see photo), however, the text is displayed as received from the API.
IMG_5871A3F577C9-1

2. Despite the code displays the genre, the film 'type' is displayed, as genre is not provided by API, but the film 'type' is.

### CODE REVIEW

File: wewatch-client/src/views/swiping/swiping-container.js [hunk 1]
```diff
@@ -21,7 +21,12 @@ const SwipingContainer = (props) => {
 
   const [index, setIndex]  = useState(0);
   const [title, setTitle] = useState(topTenMovies[index].title);
-  const [imageURL, setImageUrl] = useState(topTenMovies[index].image)
+  const [imageURL, setImageUrl] = useState(topTenMovies[index].image);
+  const [year, setYear] = useState(topTenMovies[index].released);
+  const [lengthOfMovie, setLengthOfMovie] = useState(topTenMovies[index].duration);
+  const [rating, setRating] = useState(topTenMovies[index].rating);
+  const [genre, setGenre] = useState(topTenMovies[index].type);
```
Comment (rosakng) [inline wewatch-client/src/views/swiping/swiping-container.js:28]: Maybe you can just change this variable name to typeOfMovie and name the function setTypeOfMovie to coincide with the movie type

File: wewatch-client/src/views/swiping/swiping-container.js [hunk 2]
```diff
@@ -89,12 +99,12 @@ const SwipingContainer = (props) => {
...
             <MovieDetail
               title={title}
-              year="2016"
-              lengthOfMovie="2h 28m"
-              rating="8/10"
-              genre="Thriller"
+              year={year}
+              lengthOfMovie={lengthOfMovie}
+              rating={rating}
+              genre={genre}
```
Comment (rosakng) [inline wewatch-client/src/views/swiping/swiping-container.js:105]: What are your thoughts on having a label “Media Type: “ with the type beside it? So the field is clearer to the user. Let me know what you think!

File: wewatch-client/src/views/swiping/swiping-container.js [hunk 3]
```diff
@@ -89,12 +99,12 @@ const SwipingContainer = (props) => {
...
               imageURL={imageURL}
-              description="Inception is a 2010 science fiction action film written and directed by Christopher Nolan, who also produced the film with his wife, Emma Thomas. The film stars Leonardo DiCaprio as a professional thief who steals information by infiltrating the subconscious of his targets."
+              description={description}
```
Comment (rosakng) [inline wewatch-client/src/views/swiping/swiping-container.js:107]: Niceee 👍

**Review by billxsheng — APPROVED** [review]
Looks good! Rosa addressed all the changes I was thinking of.

File: wewatch-client/src/views/swiping/swiping-container.js (hunk 1 above)
Comment (varoongupta) [reply wewatch-client/src/views/swiping/swiping-container.js:28]: Yeah, that makes more sense, changes pushed!

File: wewatch-client/src/views/swiping/swiping-container.js (hunk 2 above)
Comment (varoongupta) [reply wewatch-client/src/views/swiping/swiping-container.js:105]: Agreed, resolved with the previous comment. Thanks!

**Review by rosakng — APPROVED** [review]
LGTM!!

**Pull request merged**